doc/examples/applications/plot_3d_segmentation_embryo.py

```diff
@@ -0,0 +1,206 @@
+"""
+==================================================
+Segment 3D image sample of developing mouse embryo
+==================================================
+
+In this example, we look at a microscopy image of a developing mouse embryo.
+We use sample data from [1]_, more precisely from embryo B at time point 184.
+
+.. [1] McDole K, Guignard L, Amat F, Berger A, Malandain G, Royer LA,
+       Turaga SC, Branson K, Keller PJ (2018) "In Toto Imaging and
+       Reconstruction of Post-Implantation Mouse Development at the
+       Single-Cell Level" Cell, 175(3):859-876.e33.
+       ISSN: 0092-8674
+       :DOI:`10.1016/j.cell.2018.09.031`
```

Comment:
Not sure if that's supported but what do you think about moving this out of the way to the end of the example?